kie-dmn-core/src/main/java/org/kie/dmn/core/compiler/DMNCompilerImpl.java

```diff
@@ -142,9 +142,22 @@ private void processDrgElements(DMNCompilerContext ctx, DMNFEELHelper feel, DMNM
         for ( BusinessKnowledgeModelNode bkm : model.getBusinessKnowledgeModels() ) {
             BusinessKnowledgeModelNodeImpl bkmi = (BusinessKnowledgeModelNodeImpl) bkm;
             linkRequirements( model, bkmi );
-            FunctionDefinition funcDef = bkm.getBusinessKnowledModel().getEncapsulatedLogic();
-            DMNExpressionEvaluator exprEvaluator = evaluatorCompiler.compileExpression( ctx, model, bkmi, bkm.getName(), funcDef );
-            bkmi.setEvaluator( exprEvaluator );
+
+            ctx.enterFrame();
+            try {
+                for( DMNNode dep : bkmi.getDependencies().values() ) {
```

Comment:
"linking" the dependencies similarly to what happens below for `DecisionNode`s as well